packages/css-parser-algorithms/test/cases/source-indices/0001.mjs

```diff
@@ -0,0 +1,81 @@
+import { tokenize } from '@csstools/css-tokenizer';
+import assert from 'assert';
+import { parseCommaSeparatedListOfComponentValues, sourceIndices } from '@csstools/css-parser-algorithms';
+
+const onParseError = (err) => {
+	throw err;
+};
+
+{
+	const indices = [
+		[52, 65],
+		[51, 51],
+		[51, 65],
+		[45, 48],
+		[44, 44],
+		[43, 43],
+		[39, 40],
+		[38, 38],
+		[37, 37],
+		[36, 36],
+		[35, 35],
+		[30, 41],
+		[29, 29],
+		[16, 27],
+		[12, 28],
+		[11, 11],
+		[1, 42],
+		[0, 49],
+		[0, 49],
+	];
+
+	const parts = [
+		'something-else',
+		' ',
+		' something-else',
+		'50px',
+		' ',
+		',',
+		'1x',
+		' ',
+		'*',
+		' ',
+		'2',
+		'calc(2 * 1x)',
+		' ',
+		'"image1.jpg"',
+		'url("image1.jpg")',
+		' ',
+		'image-set( url("image1.jpg") calc(2 * 1x))',
+		'(image-set( url("image1.jpg") calc(2 * 1x)), 50px)',
+		'(image-set( url("image1.jpg") calc(2 * 1x)), 50px)',
+	];
```

Comment:
Since you always need both, might make sense to put them together like
```js
const indicesAndParts = [
  [[52, 65], 'something-else'],
  ...
];
...
const [indices, part] = indicesAndParts.pop();
```
or even
```js
const indicesAndParts = [
  [52, 65, 'something-else'],
  ...
];
...
const [first, last, part] = indicesAndParts.pop();
// Pass as [first, last] 
```
and maybe call `Array.prototype.reverse` on the array so you can define them in order.

Reply:
Thank you @ehoogeveen-medweb 🙇 

----

Are you interested in a getting more access to this project?
I really value that you take the time to review pull requests like these.

If you are interested we could make you a member of this repository?

Reply:
I can't promise that I'll be very involved (busy busy!), I mostly tend to follow conversations and leave drive-by comments like this. But feel free to ping me if you'd like input on something, and feel free to add me if you want me to leave actual review comments :) (I assume you need to be a member to do that)

Reply:
Neither I nor any of the other maintainers would expect any real commitment from you :)

The main difference is that you would have more tools available :
- approve/reject pull requests
- close/re-open issues
- create a branch and a pull request without a fork

But following and commenting as you have been doing is also absolutely fine!  
Completely up to you :)